test/clients.go

```diff
@@ -30,12 +30,27 @@ import (
 
 // Clients holds instances of interfaces for making requests to Knative Serving.
 type Clients struct {
-	Kube      *kubernetes.Clientset
+	KubeClient    *KubeClient
```

Comment:
will kubeclient be the only one to be moved to knative/pkg?

Reply:
yes. kubeclient will be moved to pkg. buildclient is currently not consumed by build tests. Will be keeping it in serving only.